bluetooth/comparison_w_volume/front_30minagg.sql

```diff
@@ -1,16 +1,39 @@
-﻿SELECT bt.thirtymins as datetime_bin, bt.obs, mio.volume  
+﻿SELECT
+    bt.thirtymins AS datetime_bin,
+    bt.obs,
+    mio.volume
 
 FROM (
-	SELECT intersection_uid, date_trunc('hour', datetime_bin)+ date_part('minute', datetime_bin)::int / 30 * interval '30 min'  as thirtymins, leg, dir, sum(volume) as volume 
-	FROM miovision.volumes_15min
-	WHERE (intersection_uid = 6  AND leg = 'W'AND dir = 'EB' 
-	AND datetime_bin::date in ('2017-11-01', '2017-11-02', '2017-11-03','2017-11-06','2017-11-07') AND classification_uid in (1,4,5))
-	GROUP BY intersection_uid, thirtymins, leg, dir) mio, 
+    SELECT
+        intersection_uid,
+        leg,
+        dir,
+        date_trunc('hour', datetime_bin)
+        + date_part('minute', datetime_bin)::int / 30 * interval '30 min' AS thirtymins,
+        sum(volume) AS volume
+    FROM miovision.volumes_15min
+    WHERE (
+        intersection_uid = 6 AND leg = 'W' AND dir = 'EB'
+        AND datetime_bin::date IN (
+            '2017-11-01', '2017-11-02', '2017-11-03', '2017-11-06', '2017-11-07'
+        )
+        AND classification_uid IN (1, 4, 5)
+    )
+    GROUP BY intersection_uid, thirtymins, leg, dir
+) AS mio,
 
-	(SELECT date_trunc('hour', datetime_bin)+ date_part('minute', datetime_bin)::int / 30 * interval '30 min'  as thirtymins, sum(obs) as obs 
-	FROM bluetooth.aggr_15min
-	WHERE analysis_id = 1454523
-	AND datetime_bin::date in ('2017-11-01', '2017-11-02', '2017-11-03','2017-11-06','2017-11-07')
-	GROUP BY thirtymins) bt
+    (
```

Comment:
do we need to get rid of these fluffing checks too?

Reply:
It's not able to auto-fix everything. In particular, it does seem to have problems indenting sub-selects - I think because it would prefer they were refactored as CTEs in most cases.